.changelog/2304.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+helm: Kubernetes v1.27 is now supported. Minimum tested version of Kubernetes is now v1.24.
```

Comment:
This didn't really require a code change to Consul-K8s, but I noticed we added a line like this for 1.26 last release... so did that here as well